### PR TITLE
Install child folder deps through helper script

### PR DESCRIPTION
**Pull request by paulirish**
in #486 we discovered that the `npm install -g GoogleChrome/lighthouse` is failing. 
It appears to fail because the postinstall script runs before the `npm --prefix` will actually work.
As a result the -core and -extension installs are a noop.

This PR changes the postinstall to use a small helper script. (It's in JS, though it would work the same in bash).

In the case that install fails, we'll see something like this in the terminal:

image

The user can then run that command manually, and it will work.
(I can't say I love this solution, but I cannot find another mechanism to get `npm install -g GoogleChrome/lighthouse` working.)

### CODE REVIEW

File: _install_deps.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const exec = require('child_process').exec;
```
Comment (paullewis) [inline _install_deps.js]: Needs a license.

File: _install_deps.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+var cwd = require('path').resolve();
```
Comment (paullewis) [inline _install_deps.js]: `cwd` doesn't seem to get overwritten, should be `const`.

File: _install_deps.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+const corePath = `${cwd}/lighthouse-core`;
+const extPath = `${cwd}/lighthouse-extension`;
+
+const npm = 'npm --prefix';
+const cmd = `${npm} ${corePath} install ${corePath} && ${npm} ${extPath} install ${extPath}`;
+
+console.log(cmd);
```
Comment (paullewis) [inline _install_deps.js]: Do we need a log here? If so might be worth documenting what it is.

**Comment by paulirish** [comment]
updated. ptal

**Comment by paullewis** [comment]
A+ would review again

**Pull request merged**